src/joystick/iphoneos/SDL_mfijoystick.m

```diff
@@ -180,6 +180,10 @@ @interface GCMicroGamepad (SDL)
 static BOOL
 IOS_AddMFIJoystickDevice(SDL_JoystickDeviceItem *device, GCController *controller)
 {
+    if ((@available(macOS 11.3, *)) && !GCController.shouldMonitorBackgroundEvents) {
```

Comment:
Is there a point to the `!GCController.shouldMonitorBackgroundEvents` check before setting it?

Reply:
Yes, if the assignment does more work under the hood. I don't know the implementation here, so I don't know whether that's true in this case.

Reply:
My only reasoning was not to chance anything (especially after finding the initial GCControllerDidConnectNotification requirement to enable it undocumented), and that it doesn't need to be set more than once for the effect to persist across instances.